py/torch_tensorrt/dynamo/_exporter.py

```diff
@@ -292,12 +296,16 @@ def inline_trt_modules(
             # Insert getitem nodes as outputs (for export serialization to work)
             with gm.graph.inserting_after(trt_node):
                 getitem_output = gm.graph.call_function(operator.getitem, (trt_node, 0))
+                getitem_output.meta["val"] = trt_node.meta["val"]
```

Comment:
BERT appears to be having an issue with its `dict` output in the exporter. Is this something that could be resolved in the `output_spec` section?

Reply:
Are you referring to BERT failing test in test_models_export ?

Reply:
Yes, that is the test case I am referring to

Reply:
That's different. BERT wasn't being exported properly (torch.export.export() was failing due to some nightly torch changes). Looks like they got fixed as of today's nightly